components/engine_traits/src/tablet.rs

```diff
@@ -228,11 +228,12 @@ impl<EK> TabletRegistry<EK> {
 
     pub fn parse_tablet_name<'a>(&self, path: &'a Path) -> Option<(&'a str, u64, u64)> {
```

Comment:
> "self" is not needed?

Yes, it exists just to make calling easier. Otherwise, I need to write `TabletRegistry::<EK>::parse_tablet_name()`.

> should not expect "_" is part of the prefix.

`SPLIT_PREFIX` is defined as `split_` for simplicity.